.github/workflows/unit-tests.yaml

```diff
@@ -110,6 +110,52 @@ jobs:
           CXX: g++
           FLB_OPT: ${{ matrix.flb_option }}
 
+
+  run-qemu-aarch64-unit-tests:
+    # We chain this after Linux one as there are CPU time costs for QEMU emulation
+    needs:
+      - run-ubuntu-unit-tests
+    runs-on: ubuntu-20.04
```

Comment:
We can run on the ARM runners from CNCF using Actuated btw, we use them for building already and other projects use for testing

Reply:
Got it. I found the actuated set up line: https://github.com/fluent/fluent-bit/blob/master/.github/workflows/call-build-linux-packages.yaml#L125

The main point of using this emulated workflow is not required for budget. Plus, this workflow can be extended to use for s390x and ppc64le(PowerPC 64bit Little Endian).
We really want to confirm these platforms in the future.

Reply:
In addition, the emulated aarch64 builds are spent less than 30 minutes. It's actually less than time costs of coverage test (~50 mins)

Reply:
I think we should leave the scaffolding in place to use it for s390 as you mentioned but we should use actuated for aarch64 as Pat said.

Reply:
Budget-wise I think we're fine but keeping it simple is better I'd say so all good. Main thing was to ensure we don't block for some really long time - like the builds do if emulated!

Reply:
For blocking build perspective, it won't be a blocker even if using emulated build. On of the main candidates of the blockers is coverage task which is required for over 40 minutes long.

Reply:
Yup, personally I'm always preferring tests we can run locally as well so emulation is nice for that.